docs/donating-code.md

```diff
@@ -0,0 +1,50 @@
+# Donating Code to the CNAB Project
+
+The CNAB project has a history of collaboration across companies and countries
+on the specification and its associated code bases. The
+[CNAB GitHub organization](https://github.com/cnabio) is home to the
+specification itself as well as reference implementations and helper libraries.
+Much of the code found there was developed by one company or person and then
+donated to the CNAB project. This document details what kind of code projects
+are is accepted into the CNAB project and what the process for donating it is.
```

Comment:
"what kind of code projects _are_ accepted"

Reply:
Nice catch, thanks :)